### PR TITLE
[nginx] Update nginx to 1.10.0 and add module for sticky sessions

### PR DESCRIPTION
**Pull request by aledbf**

### CODE REVIEW

**Comment by bprashanth** [comment]
is this still wip?

**Comment by aledbf** [comment]
@bprashanth yes, I'm still testing the sticky sessions module

**Comment by aledbf** [comment]
@bprashanth now is ready

File: images/nginx-slim/rc.yaml [hunk 1]
```diff
@@ -29,6 +29,6 @@ spec:
     spec:
       containers:
       - name: nginxslim
-        image: gcr.io/google_containers/nginx-slim:0.4
+        image: gcr.io/google_containers/nginx-slim:0.7
```
Comment (bprashanth) [inline images/nginx-slim/rc.yaml]: was there a 0.6?

**Comment by bprashanth** [comment]
LGTM

**Comment by bprashanth** [comment]
will merge when you confirm that the version bump isn't an off by one error

**Comment by aledbf** [comment]
@bprashanth fixed.

**Comment by bprashanth** [comment]
LGTM

**Pull request merged**